backend/src/main/java/ca/bc/gov/hlth/hnweb/controller/AuditController.java

```diff
@@ -37,6 +42,8 @@
 @RestController
 public class AuditController extends BaseController {
 
+	private static final String DATE_TIME_FORMAT = "yyyy-MM-dd_HH-mm-ss";
```

Comment:
This isn't a super common date time format. Maybe try something like yyyy-MM-dd'T'HH:mm:ss instead?